src/Orckestra.Composer.Cart/Repositories/Order/OrderRepository.cs

```diff
@@ -125,5 +125,10 @@ public virtual Task<Overture.ServiceModel.Orders.Order> UpdateOrderAsync(UpdateO
 
             return OvertureClient.SendAsync(request);
         }
+
+        public async Task<OrderSettings> GetOrderSettings()
+        {
```

Comment:
define new cache category key OrderSettings  and use it
exapmle:
CacheConfigurationCategoryNames.cs
file \src\Orckestra.Composer.Website\App_Config\Caching.config
    <category name="ProductSettings" profileName="LongLived" />